src/containers/Application/ApplicationSubmission.tsx

```diff
@@ -65,7 +65,9 @@ const ApplicationSubmission: React.FC = () => {
     <Loading />
   ) : serialNumber && appStatus ? (
     <Segment.Group style={{ backgroundColor: 'Gainsboro', display: 'flex' }}>
-      <Header textAlign="center">{strings.TITLE_COMPANY_PLACEHOLDER}</Header>
+      <Header textAlign="center">
+        {currentUser?.organisation?.orgName || strings.TITLE_NO_ORGANISATION}
+      </Header>
```

Comment:
Hmmm perhaps we should move this surrounding to a common component?

Reply:
That'd be a pretty small component, wouldn't it? Shall we save it till we've got a few more things like this and do it more generally? 
Personally, I find this expression not too unwieldy -- it's pretty clear what it is IMHO.

Reply:
No problem. We can leave this for [Epic#10](#78). I was thinking more like the whole page, with common title.